src/pages/index.js

```diff
@@ -48,8 +48,9 @@ const IndexPage = ({data}) => {
   
   return (
     <Layout>
-      <SEO title="What does it take to decarbonize your state?" />
-
+      <SEO title="What does it take to decarbonize your state?" 
+        description={`To get to zero by 2050, the US must cut climate pollution by ${cutPerYearPrcnt}% a year.`}
```

Comment:
@derekeder - do you have any suggestions on wording for the site description?

My thought though:

> You can help decarbonize your state, and it's a lot simpler than you think! Learn how with Decarbonize My State.

Reply:
I have updated the copy. Can change it again if we get more feedback.